Classes/Command/MigrationsCommandController.php

```diff
@@ -114,7 +114,7 @@ protected function handleException(\Exception $exception)
         $message = $this->throwableStorage->logThrowable($exception);
         $this->outputLine($message);
         $this->logger->error($message, LogEnvironment::fromMethodName(__METHOD__));
-        $this->quit(1);
+        throw $exception;
```

Comment:
I presume the exit code will still be > 0?

Reply:
When I tried it (an exception was forced) the status code was 1. So yes.